src/data/styles/footer.scss

```diff
@@ -0,0 +1,25 @@
+@import "./variables.scss";
+
+footer {
```

Comment:
The `footer` tag can be used in other places besides the global footer element, and to avoid conflicts it's better to add a class to the element and style it through the class.

```
.footer {
  ...
}
```